llvm/include/llvm/CodeGen/SelectionDAGNodes.h

```diff
@@ -970,10 +970,10 @@ END_TWO_BYTE_PACK()
   static bool areOnlyUsersOf(ArrayRef<const SDNode *> Nodes, const SDNode *N);
 
   /// Return the number of values used by this operation.
-  unsigned getNumOperands() const { return NumOperands; }
+  uint32_t getNumOperands() const { return NumOperands; }
```

Comment:
Is there any reason to change this if we aren't changing the users of this function?